src/main/java/dev/vasyl/proj/model/Book.java

```diff
@@ -7,21 +7,17 @@
 import jakarta.persistence.Id;
 import jakarta.persistence.Table;
 import java.math.BigDecimal;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
 import lombok.Getter;
-import lombok.NoArgsConstructor;
 import lombok.Setter;
-import lombok.ToString;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
-@Entity
-@Table(name = "books")
 @Getter
 @Setter
-@Builder
-@NoArgsConstructor
-@AllArgsConstructor
-@ToString
+@Entity
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
```

Comment:
check if `@Where` is deprecated or not

Reply:
![image](https://github.com/user-attachments/assets/cae51c24-3a6a-42ae-b29f-022e0bf12a99)